frontend/components/person-card/index.js

```diff
@@ -23,40 +23,40 @@ const PersonCard = ({ cardDetails, bgColord, rounded, boxShadow }) => {
       {cardDetails && (
         <div
           style={{ boxShadow: boxShadow && '0 0 40px rgba(0, 0, 0, 0.1)' }}
-          className={cn('card', 'overflow-hidden', 'px-4', 'py-8', {
+          className={cn('card', 'overflow-hidden', 'px-10', 'py-10', {
```

Comment:
here you changed the global person-card padding which won't be good for both cards in our design
I suggest adding padding conditionaly on which card is the component used for